tuf/tuf.go

```diff
@@ -173,6 +173,11 @@ func (tr *Repo) RemoveBaseKeys(role string, keyIDs ...string) error {
 	return nil
 }
 
+// GetAllLoadedRoles returns a list of all role entries loaded in this TUF repo, could be empty
+func (tr *Repo) GetAllLoadedRoles() []*data.Role {
```

Comment:
I actually wouldn't expect this function to exist. As the `keysDB` has to be injected when creating a new Repo object, the caller should hold onto its reference if it needs to call `GetAllRoles`. These kinds of passthrough were what caused confusion and got us into trouble in other places.